### PR TITLE
Removing circular dependency on Helpers

### PR DESCRIPTION
**Pull request by murilommen**
details:
* models.granularity was being used on monitor_helpers, which caused issues if the full monitor package wasn't imported before importing monitor_helpers

### CODE REVIEW

File: whylabs_toolkit/helpers/monitor_helpers.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 from whylabs_toolkit.helpers.config import Config
 from whylabs_toolkit.helpers.utils import get_monitor_api, get_models_api
-from whylabs_toolkit.monitor.models import Granularity
```
Comment (naddeoa) [inline whylabs_toolkit/helpers/monitor_helpers.py:9]: If the models file has a dependency on this file then another option would be to just move the Granularity definition into its own file that they can both use, then you'd keep the type safety.

Comment (murilommen) [reply whylabs_toolkit/helpers/monitor_helpers.py:9]: it's true, I think I got a wrong commit in for some reason. fixed now

Comment (murilommen) [reply whylabs_toolkit/helpers/monitor_helpers.py:9]: on commit `702868e1852071bc7068f2073ef75d7c643a3ad8` Granularity is not there too

**Review by naddeoa — APPROVED** [review]
beautiful

**Pull request merged**